flink_jira_bot.py

```diff
@@ -65,14 +64,16 @@ def get_args():
     stale_assigned_rule = StaleAssignedRule(
         jira, jira_bot_config["stale_assigned"], args.dryrun
     )
-    stale_minor_rule = StaleMinorRule(jira, jira_bot_config["stale_minor"], args.dryrun)
-    stale_major_rule = StaleMajorOrAboveRule(
+    stale_minor_rule = StaleUnassignedRule(
+        jira, jira_bot_config["stale_minor"], args.dryrun, "Major", "Not a Priority"
+    )
+    stale_major_rule = StaleUnassignedRule(
         jira, jira_bot_config["stale_major"], args.dryrun, "Major", "Minor"
     )
```

Comment:
How is it supposed to be working? We have now two `StaleUnassignedRule`s, both from `Major` one to `Minor` other to `Not a Priority` at the same time?

Reply:
It is not ;) One of the them should be Minor. Fixed.